spec/diddocuments.md

```diff
@@ -7,26 +7,23 @@ DID document is generated algorithmically from the DID value itself, and nothing
 else. If the AID is [[ref: transferrable]], then the [[ref: KEL]] and the [[ref: TEL]]
 are also required inputs to the generation algorithm.
 
-DID documents are processable as pure JSON. Parties who wish to process them as
-JSON-LD can do so by referencing [this @context](https://example.com/context).
-
-The algorithm for transforming an AID to a DID document begins with the same
-algorithm as the did:key method. TODO: add info from TEL for service endpoints
-and other keys.
+DID documents for this method are pure JSON. They may be processed as JSON-LD by
+prepending an `@context` if consumers of the documents wish.
 
 Hashes, cryptographic keys, and signatures are represented as CESR strings. This
 is an approach similar to multibase, making them self-describing and terse.
 
-### DID Document from the KERI KEL
-The key event log [[ref: KEL]] represents a cryptographic chain
-of custody from the [[ref: AID]] itself down to the current set of signing keys and next rotation key commitments.  When generating a
-DID document an implementation must "walk the KEL" (follow the chain of key events) to determine the current key state.  Due to the fact that the KERI protocol
-allows for non-establishment event (events that don't change key state) to be intermixed in a KEL with establishment
+### DID Document from KERI Events
+The [[ref: KERI event stream]] represents a cryptographic chain
+of custody from the [[ref: AID]] itself down to the current set of signing keys and next rotation key commitments,
+as well as other data such as service endpoints.  When generating a
+DID document an implementation must "walk the [[ref: KEL]]" (follow the chain of key events) to determine the current key state.  Due to the fact that the KERI protocol
```

Comment:
Since we have discussed KERI event streams then perhaps we can avoid the new concept of 'walking the KEL' and can remain consistent about reading the stream?

Reply:
Yes I was wondering about this too while editing the spec. I wanted to replace KEL with "KERI event stream" in most places, but I wasn't sure what the equivalent expression was for "walking the KEL".